File: complexidade/javascript/higher-values.js
function higherValues(arr) {

  let newArray = new Array(arr.length).fill(0);

  for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr.length; j++) {
      if (arr[j] > arr[i]) {
        newArray[i]++;
      }
    }
  }

  return newArray;
}

const result = higherValues([7, 3, 8, 7, 5]);
console.log(result);

File: complexidade/python/higher-values.py
def higher_values(arr):
    new_array = [0] * len(arr)

    for i in range(len(arr)):
        for j in range(len(arr)):
            if arr[j] > arr[i]:
                new_array[i] += 1

    return new_array


result = higher_values([7, 3, 8, 7, 5])
print(result)
